src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/IntegrationTestRuntime.kt

```diff
@@ -1,20 +1,20 @@
 package com.atlassian.performance.tools.awsinfrastructure
 
-import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
 import com.amazonaws.regions.Regions
 import com.atlassian.performance.tools.aws.api.Aws
 import com.atlassian.performance.tools.workspace.api.RootWorkspace
 import org.apache.logging.log4j.core.config.ConfigurationFactory
+import java.util.function.Predicate
 
 object IntegrationTestRuntime {
     val taskWorkspace = RootWorkspace().currentTask
     val aws: Aws
 
     init {
         ConfigurationFactory.setConfigurationFactory(LogConfigurationFactory(taskWorkspace))
-        aws = Aws(
-            region = Regions.EU_WEST_1,
-            credentialsProvider = DefaultAWSCredentialsProviderChain()
-        )
+        aws = Aws.Builder(Regions.EU_WEST_1)
+            .availabilityZoneFilter(Predicate { it.zoneName in listOf("eu-west-1a", "eu-west-1c") })
```

Comment:
Alternatively, we can blacklist `eu-west-1b` instead of whitelisting A and C. If they add a D AZ, we'd pick it up transparently.

Reply:
> Alternatively, we can blacklist `eu-west-1b` instead of whitelisting A and C. If they add a D AZ, we'd pick it up transparently.

That's why I didn't want to blacklist. If they add a D AZ, and it will have no capacity for our instances, then our builds will start to fail again.

Reply:
OTOH if there's more capacity in D, we won't use it

Reply:
Also, if you change the region, you'll have to change the whitelist too. But a blacklist would be immune to this change.

Reply:
It's the same problem that we have with other dependencies. We can have new features and bug fixes in the latest versions, but we also can get a new bug. We choose to lock dependencies for our builds to make them repeatable and not to disturb our work. Each time we bump dependencies we do it consciously. IMO we should stick to this strategy also for infrastructure dependencies.

Reply:
Fewer moving parts, fair enough 👍